internal/generator/vector/output/otlp/transform.go

```diff
@@ -28,7 +28,8 @@ resource.attributes = append( resource.attributes,
 resource.attributes = append( resource.attributes,
     [{"key": "k8s.pod.name", "value": {"stringValue": get!(.,["kubernetes","pod_name"])}},
     {"key": "k8s.container.name", "value": {"stringValue": get!(.,["kubernetes","container_name"])}},
-    {"key": "k8s.namespace.name", "value": {"stringValue": get!(.,["kubernetes","namespace_name"])}}]
+    {"key": "k8s.namespace.name", "value": {"stringValue": get!(.,["kubernetes","namespace_name"])}},
+    {"key": "k8s.stream", "value": {"stringValue": get!(.,["kubernetes","stream"])}}]
```

Comment:
For OTLP output, if you are adding to the resource attributes, then it also needs to be included in the `group_by` for container logs.  https://github.com/openshift/cluster-logging-operator/blob/master/internal/generator/vector/output/otlp/group_by.go#L39